arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}": {
+      "get": {
+        "operationId": "DatabaseAccounts_Get",
+        "description": "Retrieves the properties of an existing Azure DocumentDB database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The database account properties were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccount"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "DatabaseAccounts_Patch",
+        "description": "Patches the properties of an existing Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "newTags",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountPatchParameters"
+            },
+            "description": "The tags parameter to patch for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The properties of the database account were patched successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccount"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "DatabaseAccounts_CreateOrUpdate",
+        "description": "Creates or updates an Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```

Comment:
This was done as per the ARM standard mentioned in the Resource Provider API v2 documentation which mentioned that a 200/201 would be sufficient as long as the provisioningState property gets populated and updated. And this also works with the current portal.

For the time being, would it be better to remove the long-running-operation tag?